dev-server/static/scripts/util.js

```diff
@@ -19,7 +19,9 @@ export function loadClient(clientUrl) {
  * This uses the same method as the browser extension does to deactivate the client.
  */
 export function unloadClient() {
-  const annotatorLink = document.querySelector('link[type="application/annotator+html"]');
-  annotatorLink?.dispatchEvent(new Event('destroy'));
+  const annotatorLink = document.querySelector('link[type="application/annotatror+html"]');
+  // Do not use optional chaining (?.) below. Because this code is not bundled by babel
```

Comment:
I'd suggest to add a general note at the top of the file that it is not transpiled, so future readers are aware, then remove this more specific comment.

Reply:
Good idea.